components/Cards/Card.js

```diff
@@ -0,0 +1,51 @@
+import SingleIconButton from "../Buttons/IconsButton"
+
+export default function Card(props) {
+    const { name, image, description } = props
+    return (
+        <>
+            <div className="item-shadow h-100 d-flex flex-column justify-content-between">
+                <div>
+                    <div
+                        className="img-box"
+                        style={{ backgroundImage: `url(${image})` }}
+                    ></div>
+                    <div className="p-3">
+                        <div>
+                            <h5 className="text-center">{name}</h5>
+                        </div>
+                        <div>
+                            <br></br>
+                            <h4>Contributed to:</h4>
+                            <a
+                                href="https://github.com/COPS-IITBHU/hackalog-frontend"
+                                className="text-muted"
+                            >
+                                {description[0]}
+                            </a>
+                            <br></br>
+                            <a
+                                href="https://github.com/COPS-IITBHU/hackalog-backend"
+                                className="text-muted"
+                            >
+                                {description[1]}
+                            </a>
+                        </div>
+                    </div>
+                </div>
+                <div className="text-center py-3">
+                    <a href={props.github} target="_blank">
+                        <SingleIconButton>Github Profile</SingleIconButton>
```

Comment:
How about we show the GitHub username instead of "Github Profile" here? The only change required would be to pass the username in the props.

Reply:
ok, I'll do so